csharp/ql/src/semmle/code/csharp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -2553,12 +2726,10 @@ private module FlowExploration {
 
     override Configuration getConfiguration() { result = config }
 
-    private PartialPathNodePriv getSuccMid() {
+    override PartialPathNodePriv getASuccessor() {
```

Comment:
This rewrite exposes the otherwise private class `PartialPathNodePriv`.

Reply:
Still, the class can't be used, so I think it's OK.

Reply:
It's members can now be accessed.  I had to write a small test to check that this was indeed the case - I don't know whether that should be considered a bug in QL.

Reply:
I would consider it a bug in QL :-)